serialization/library_version_type.hpp: No corresponding #pragma warning(pop)

Problem has appeared in 1.74.0:
`...boost/serialization/library_version_type.hpp(28,18): warning C5032: detected #pragma warning(push) with no corresponding #pragma warning(pop)`

https://github.com/boostorg/serialization/blob/e94aebed7a0b9e0fc7e4d3d516be3c5482b2a377/include/boost/serialization/library_version_type.hpp#L28

--- a/include/boost/serialization/library_version_type.hpp
+++ b/include/boost/serialization/library_version_type.hpp
@@ -65,6 +65,10 @@ public:
     }
 };
 
+#if defined(_MSC_VER)
+#pragma warning( pop )
+#endif
+
 } // serialization
 } // boost
 

